Demo/src/main/java/com/braintreepayments/demo/PayPalRequestFactory.java

```diff
@@ -118,17 +123,124 @@ public static PayPalVaultRequest createPayPalVaultRequest(
         return request;
     }
 
+    private static Float calculateItemTotal(
+            String amount,
+            String taxTotal,
+            String shippingTotal,
+            String handling,
+            String insurance,
+            String discountTotal,
+            String shippingDiscount
+    ) {
+        float tax = Float.parseFloat(taxTotal);
+        float shipping = Float.parseFloat(shippingTotal);
+        float handlingFee = Float.parseFloat(handling);
+        float insuranceFee = Float.parseFloat(insurance);
+        float discount = Float.parseFloat(discountTotal);
+        float shippingDisc = Float.parseFloat(shippingDiscount);
+        float total = Float.parseFloat(amount);
+
+        float additions = tax + shipping + handlingFee + insuranceFee;
+        float subtractions = shippingDisc + discount;
```

Comment:
The naming here is a bit confusing, maybe something like this would make it slightly more clear?
```suggestion
        float additionalFees = tax + shipping + handlingFee + insuranceFee;
        float discountsFromFees = shippingDisc + discount;
```